File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/go/valid.go
// Package fixture — café 日本語 before declarations
package fixture

import "fmt"

// Greeter greets.
type Greeter struct {
	Name string
}

// Hello says hello.
func (g Greeter) Hello() string {
	return fmt.Sprintf("hi %s", g.Name)
}

// Shape is an interface.
type Shape interface {
	Area() float64
}

type ID = string

const Limit = 10

var counter int

func helper(x int) int { return x + 1 }

File: benchmarks/stage5_vexp_swe_bench_smoke/fixtures/m202_language_families/swift/valid.swift
// Fixture — café 日本語 before declarations
import Foundation

/// A greeter.
class Greeter {
    /// Say hello.
    func hello(name: String) -> String { return "hi \(name)" }
    init() {}
}

struct Point { var x: Int }

protocol Shape {
    func area() -> Double
}

enum Color { case red, blue }

typealias Identifier = String

func helper(x: Int) -> Int { return x + 1 }
